org.sebicodes.Prt.prt.yml

```diff
@@ -0,0 +1,19 @@
+app-id: io.github.Prt.prt
```

Comment:
You don't seem to be user `prt`.

Reply:
oh, so prt has to be my username or Prt?

Reply:
it's `io.github.` ORGINAZATION/USER `.` APPNAME